tests/e2e_tests/multistep/test_axon.py

```diff
@@ -109,4 +129,29 @@ async def test_axon(local_chain):
     assert updated_axon.port == 8091
     assert updated_axon.hotkey == alice_keypair.ss58_address
     assert updated_axon.coldkey == alice_keypair.ss58_address
+
+    # Terminate miner and make sure nothing bad happened.
+    if axon_process.returncode == None:
+        logging.info("terminating miner")
+        try:
+            # try/except to prevent exception in case the process just ends now
+            axon_process.terminate()
+        except:
```

Comment:
Let's not do bare `except`s. Ideally, we should know what we're trying to catch, but if not, at least `except Exception as e` and log it.